Sources/Testing/Events/Event.swift

```diff
@@ -198,13 +223,16 @@ public struct Event: Sendable {
   ///     ``Test/Case/current``.
   ///   - instant: The instant at which the event occurred. The default value
   ///     of this argument is `.now`.
+  ///   - sourceLocation: The source location at which the event occurred, if
+  ///   	known, or `nil` if that information is not applicable.
```

Comment:
Funky whitespace

Reply:
That's what I get for using my personal computer.